x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/components/add_domain/validation_step_panel.test.tsx

```diff
@@ -44,7 +44,7 @@ describe('ValidationStepPanel', () => {
           action={action}
         />
       );
-      expect(wrapper.find('[data-test-subj="errorMessage"]').dive().text()).toContain(
+      expect(wrapper.find('[data-test-subj="errorMessage"]').childAt(0).text()).toContain(
```

Comment:
Not sure if this is the best way to do it but the specs were failing with `dive()`.

Reply:
This reminds me that I perhaps should add a new spec with a scenario where a URL is turned into a link.